src/parser/parser_driver.cpp

```diff
@@ -36,13 +36,14 @@ void ParserDriver::defineTokens()
 		currentLocation().addColumn(str.length());
 	});
 
-	_parser.token("\n").action([&](std::string_view str) -> Value {
+	_parser.token("[\n\r]").action([&](std::string_view str) -> Value {
+		currentTokenStream()->setNewLineChar(std::string{str}.front());
```

Comment:
Tip: `std::string_view` has also [`front()`](https://en.cppreference.com/w/cpp/string/basic_string_view/front), so there is no need to convert the string view to a string.